Source/DafnyDriver/Commands/GenerateTestsCommand.cs

```diff
@@ -123,6 +124,9 @@ private enum Mode {
     dafnyOptions.TestGenOptions.Mode = mode;
   }
 
+  public static readonly Option<bool> IgnoreWarnings = new("--ignore-warnings",
+    "Ignore warnings in error messages.");
```

Comment:
```suggestion
    "Ignore warnings when generating tests.");
```

The previous text seemed contradictory to me ("warnings in error messages").